server/athenian/api/openapi/openapi.yaml

```
@@ -1808,6 +1808,10 @@ components:
         * `pr-closed`: number of closed PRs.
         * `pr-done`: number of released, closed unmerged, and force push dropped PRs.
         * `pr-size`: average size of PRs (added + deleted lines) which are active on the time interval after applying the quantiles.
+        * `pr-wip-pending-count`: number of PRs currently in Work In Progress stage
```

```suggestion
        * `pr-wip-pending-count`: number of PRs currently in Work In Progress stage.
```
(trailing dots for all the four)